docs/containers-storage.conf.5.md

```diff
@@ -93,13 +100,6 @@ until all of the entries have been used for maps.
       remap-user = "storage"
       remap-group = "storage"
 
-ostree_repo = ""
-   If specified, use OSTree to deduplicate files with the overlay backend.
-
-skip_mount_home = "false"
-   Set to skip a PRIVATE bind mount on the storage home directory.
-Only supported by certain container storage drivers (overlay).
-
 [storage.options.thinpool]
 
 Storage Options for thinpool
```

Comment:
I know it's not yours, but can you remove line 103 and then add three pounds  signs in front of 105?

'### Storage Options for thinpool`

If you don't want to deal with it, no worries, I'll follow up and fix it.

Reply:
> If you don't want to deal with it, no worries, I'll follow up and fix it.

If I'm on it, I can also tie up loose ends.